frontend/src/metabase/env.js

```diff
@@ -1,5 +1,7 @@
 export const isCypressActive = !!window.Cypress;
 
+export const isStorybookActive = !!window.STORYBOOK_ENV;
```

Comment:
I couldn't find any reference for this env. It seems storybook has [`process.env.STORYBOOK`](https://github.com/storybookjs/storybook/blob/8e51bf9b2452c501c18bafb824b4f48a57e824b3/code/core/src/common/utils/envs.ts#L18) as recommended in [this comment.](https://github.com/storybookjs/storybook/issues/16343#issuecomment-942923074)

Should we use that instead?

I think using one from window feels kind of fragile.